docs/welcome.md

```diff
@@ -1,54 +1,59 @@
-sidebar_label: Introduction
+title: Amplication Documentation
+description: Learn how Amplication helps teams build standardized backend services while enforcing best practices and preventing technical debt.
+sidebar_label: Overview
 slug: /
 ---
 
-# Get Started with Amplication
+# Amplication Documentation
 
-Welcome to Amplication's developer documentation!
+Amplication helps you build, maintain, and scale backend services that align with your organization's best practices, supporting you throughout the entire development process.
+Our platform combines live templates, AI-powered code generation, and technical debt management features to help teams create production-ready Node.js and .NET backend services while enforcing organizational standards.
 
-Amplication is an open-source, AI-powered code generation solution for rapid .NET and Node.js development. Unlike other AI code generation tools, Amplication is the only one that guarantees production-ready code generation for all your backend needs.
-
-## Quick Start
+## Quickstart
 
 [Create your first service](/first-service/) with Amplication's AI, Jovu, and generate a .NET or Node.js _production-ready codebase in minutes_.
 
-## Customize Your Generated Service
+## AI-powered Backend Generation
 
-Amplication transforms your ideas into production-ready code in minutes, but we're more than just a code generator. Our platform supports you throughout the entire development process.
+Our AI assistant Jovu helps you rapidly generate production-ready backend services with built-in best practices. Generate complete service infrastructure including REST and GraphQL APIs, database models, authentication, and logging in minutes. Customize your services through our platform's intuitive interface while maintaining consistent architecture patterns and organizational standards.
 
-Use these features to fine-tune and customize your service to your exact needs:
+- **[Jovu](/amplication-ai/)**: Generate production-ready .NET or Node.js code, get intelligent suggestions for your service's architecture, and more with Amplication AI.
+- **[Full Stack Code Generation](/getting-started/)**: Fully-functional REST and GraphQL APIs, database models, authentication, logging, React Admin UI, and more.
+- [Set up Entities and Fields](/set-up-entities)
+- [Configure API Endpoints](/custom-types-and-actions)
+- [Add Authentication and Authorization](/configure-roles-and-permissions)
 
-- [Set up entities and fields](/set-up-entities/)
-- [Configure roles and permissions](/configure-roles-and-permissions/)
-- [Create and manage custom API endpoints and data structures](/custom-types-and-actions/)
-- [Extend capabilities with plugins](/add-plugins-service/)
-- [Commit changes and build](/commit-and-build-new-versions/)
+## Embed Your Organization's Best Practices
 
-## Key Features
+Define and maintain standardized service architectures through live templates and private plugins.
```

Comment:
It's not only about the architecture- but also about code conventions, tools, etc. I think that mentioning architecture here is a little confusing